.github/workflows/gnoland.yml

```diff
@@ -50,11 +50,6 @@ jobs:
         go-version:
           - "1.20.x"
           - "1.21.x"
-        args:
-          - _test.gnoland
-          - _test.gnokey
-          - _test.pkgs
```

Comment:
Although it will resolve the issue, it won't achieve parallelization as previously (which may lead to significantly slower performance). Moreover, the tests summary won't readily pinpoint the problem, and the efficiency of re-running failed tests will be compromised. While it's a solution, it also comes with its own set of drawbacks.

I'm okay to go in this direction temporarily but we should ideally come back to per-flavor tests later if possible.

Reply:
I'm working on refining the codecov configuration on my fork. You can view the progress here: https://github.com/moul/gno.

Reply:
Running subsets of tests like that implies having a risk of not executing all of them like it was happening.

I recommend for parallelization to do it at Go level using `t.Parallell()`

Reply:
I concur. We can implement a technique akin to the one in tm2 to ensure nothing is overlooked. You can reference it here: https://github.com/gnolang/gno/blob/master/tm2/Makefile#L46.

While having t.Parallel functioning optimally is a goal, I believe it's still valuable to maintain diverse testing approaches even with t.Parallel. 

As an added suggestion, we could introduce a `go test ./...` in the master branch. As long as it's not triggered for every PR, it seems like a logical step.